### DISCUSSION TITLE
Httpd with openLDAP

### DISCUSSION THREAD

**Discussion by axelbrunger** [Homebrew/discussions#2912] [Everyday usage]
I'm trying to compile httpd with openLDAP support. Is it possible to do this with Brew?

**Comment by carlocab** [comment]
You can use `brew extract` to create your own version of the `httpd` formula with `openldap` support. Alternatively, you can make a pull request to add this to the existing formula.

If you choose to maintain your own `httpd`, these docs might help:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

If you make a pull request against the existing formula, please include an explanation in your pull request why `openldap` support is useful, and whether/how most users might be interested in making use of it.

Feel free to ask if you have any further questions.

**Reply by axelbrunger** [reply]
Thank you for the suggestions. It seems that it would be of general interest to have a formula that includes openldap, especially since Mac OS has recently discontinued php support and their Apache implementation is rather unstable. Where shall I make the pull request?

**Reply by nandahkrishna** [reply]
You'll have to create a pull request at [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core). Do take a look at these links for information on how to open a Homebrew PR:

https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md
https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request
